module/zcommon/zfs_fletcher.c

```diff
@@ -829,4 +916,6 @@ EXPORT_SYMBOL(fletcher_4_native_varsize);
 EXPORT_SYMBOL(fletcher_4_byteswap);
 EXPORT_SYMBOL(fletcher_4_incremental_native);
 EXPORT_SYMBOL(fletcher_4_incremental_byteswap);
+
+EXPORT_SYMBOL(fletcher_4_abd_ops);
```

Comment:
There shouldn't be any need to make this symbol visible outside the zcommon module.  So this can be removed.